### ISSUE TITLE
Need to add after property in Table Columns options ?

### ISSUE THREAD

**Issue by sujitkv92** [typeorm/typeorm#6194]
Can we have 'after' property in columns options  (TableColumnOptions interface) in next version so that we can add column after specific column ?

Current structure - 
await queryRunner.addColumns("tablename", [new TableColumn({
            name: "column_name",
            type: "varchar",
            isNullable: true,
            default: null,
        }),

I want to support as -
await queryRunner.addColumns("tablename", [new TableColumn({
            name: "column_name",
            type: "varchar",
            isNullable: true,
            default: null,
            after: 'existing_column_name'
        }),

**Comment by imnotjames** [comment]
Duplicate of #6167
